packages/nimble-components/src/table/styles.ts

```diff
@@ -186,6 +186,26 @@ export const styles = css`
     .row {
         position: relative;
     }
+
+    .accessibly-hidden {
+        ${
```

Comment:
This styling has been copied & pasted a number of times throughout our codebase, so I think it warrants a shared style sourced from a single location. I plan to do that in a follow up PR.

Reply:
Yep, that was my thought as well 👍